starter-code/css/main.css

```diff
@@ -7,4 +7,98 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-image : linear-gradient(to bottom, #1c5380 223px, #fff 203px);
 }
+
+.container{
+ 
+  display: flex;
+  justify-content: space-around;
+  width: 70%;
+  margin: 30px auto;
+}
+
+.img-left{
+  
+  padding: 20px;
+}
+
+.right{
+  
+  padding: 20px;
+}
+
+header{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 50px;
+  margin-bottom: 15px;
+  padding: 0 9px;
+}
+
+.logo-insta{
+  height: 100%;
+  width: auto;
+}
+
+.image-appli{
+  display: flex;
+  justify-content: center;
+  margin-top: 25px;
+}
+
+.img-space{
+  margin: 0 10px;
+}
+
+h2{
+  text-align: left;
+  color: #06365f;
+  margin-top: 0;
+}
+
+.paragraph{
+
+  background-image: url("../images/frame.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position: top center;
+  padding: 35px;
+}
+
+bouton{
```

Comment:
The bouton element doesn't exist, there only is a `button`!
Using the wrong tag, you actually created an element that you could style, but it doesn't _behave_ like a button, so you should avoid using custom tags.
I assume this is just a typo error :)